cmd/lncli/types.go

```diff
@@ -11,7 +12,9 @@ type OutPoint string
 
 // NewOutPointFromProto formats the lnrpc.OutPoint into an OutPoint for display.
 func NewOutPointFromProto(op *lnrpc.OutPoint) OutPoint {
-	return OutPoint(fmt.Sprintf("%s:%d", op.TxidStr, op.OutputIndex))
```

Comment:
are we certain this doesn't break existing rpcs?

Reply:
I went through all of the RPCs that return an `OutPoint` and they all set the bytes.

Reply:
just checked, looks like listunspent sets both so we're good